### PR TITLE
[modify] Remove unused

### PR DESCRIPTION
**Pull request by ghost**
- Error with cipher as 'AES-256-CBC'
  - [ ] **(no need)** change Encryption Key: cipher to 'AES-128-CBC' 
- Run phpunit with version of composer
  - [x] Add script command 'composer test' to run phpunit
- Remove unused
  - [x] Remove unused libs in TodoControllerTest.php, TodoData.php, TodoModelMock.php
  - [x] Delete unused file : TodoModelMock.php, TodoServiceMock.php, TodoServiceTestSeeder.php

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,10 @@
         ],
         "post-create-project-cmd": [
             "php artisan key:generate"
-        ]
+        ],
```
Comment (kondo-takuto-mulodo) [inline composer.json:48]: @luong-phuc-mulodo what's the update for?

File: config/app.php [hunk 2]
```diff
@@ -80,7 +80,7 @@
 
     'key' => env('APP_KEY', 'SomeRandomString'),
```
Comment (kondo-takuto-mulodo) [inline config/app.php:82]: @luong-phuc-mulodo why did you change cipher 256 to 128?

File: composer.json (hunk 1 above)
Comment (ghost) [inline composer.json:48]: we can run "composer test" to run phpunit by version of phpunit what you installed in composer. it doesn't depend on phpunit version what you installed in PC.

Comment (kondo-takuto-mulodo) [inline composer.json:48]: @luong-phuc-mulodo i understood. thanks.

**Comment by kondo-takuto-mulodo** [comment]
@luong-phuc-mulodo thanks for your revised. I would like to merge this branch to the master.
